README.md

```diff
@@ -23,6 +24,29 @@
   - [Contributing](#contributing)
   - [Support](#support)
 
+## Features
+
+FormSG is a form builder application built, open sourced and maintained by the [Open Government Products](https://open.gov.sg) team of the Singapore [Government Technology Agency](https://tech.gov.sg) to digitise paper processes.
+
+Notable features include:
+
+- 19 different form field types, including attachments, tables, email and mobile
+- Verified email and mobile phone fields via integrations with Twilio and AWS SES
+- Automatic emailing of submissions for forms built with Email Mode
+- End-to-end encryption for forms built with Storage Mode
+- (Singapore government agencies only) Citizen authentication with [SingPass](https://www.singpass.gov.sg/)
```

Comment:
should these link to the About pages rather than the login pages?

Reply:
agree

Reply:
Use can reuse the links I used in the deployment md